### ISSUE TITLE
postgres pipline query

### ISSUE THREAD

**Issue by MarcFaussurier** [eddic/fastcgipp#95]
Hello, 2 days ago, async-batch patch aka. pipline mode was added to postgresql master, this what allow ultra high performances for both lithium and drogon on techempower, excpect that they were using an old fork of libpq. Now that this feature is beeing added officially, maybe implementing it would be a good idea ? 

See  this commit https://github.com/postgres/postgres/commit/acb7e4eb6b1c614c68a62fb3a6a5bba1af0a2659  

old doc about that feature : https://2ndquadrant.github.io/postgres/libpq-batch-mode.html 

see drogon issue: https://github.com/an-tao/drogon/issues/526 

Or maybe i'm wrong once again, let me know.

I can help you working on it.

Thank you really much for your super nice repo. Deserve more stars.

**Comment by eddic** [comment]
Yeah this would fit very nicely into the way fastcgi++ currently does it's PostgreSQL business. The [Connection](https://fastcgipp.isatec.ca/classFastcgipp_1_1SQL_1_1Connection.html) class implements it's own queue of sorts and this could effectively replace it. We'll leave this issue open as a reminder.
